src/preview/Preview.tsx

```diff
@@ -161,7 +162,9 @@ const SlideObjectPreview = ({
         </div>
       ) : object.kind === "image" ? (
         <img
-          src={object.url}
+          src={object.src && object.src.kind === 'data'
```

Comment:
Small nitpick 😁 (applies to the next two lines too)

```suggestion
          src={object.src?.kind === 'data'
```

Reply:
No, I went to bed thinking about this as well. 👍